.circleci/config.yml

```diff
@@ -154,6 +154,7 @@ jobs:
   publish-rc-images:
     <<: *base-docker-job
     environment:
+      DOCKER_REGISTRY: osbapublicacr.azurecr.io/
       DOCKER_REPO: microsoft/
```

Comment:
With this set, the final URL will be `osbapublicacr.azurecr.io/microsoft/azure-service-broker:<tag>`, which looks awkward.

I think you probably want to unset this env var so that the final URL for the image will be `osbapublicacr.azurecr.io/azure-service-broker:<tag>`

fwiw... this env var should remain settable, just unset. The reason for this is it allows for overriding the final image URL if people want to build customized versions from their own forks and publish to somewhere else.

Reply:
Hmm.. I still want to keep "microsoft". With the change as you suggested "DOCKER_REPO" -> "DOCKER_REGISTRY_NAMESPACE", it looks good to me. How do you feel?